libraries/Microsoft.Bot.Builder.AI.Orchestrator/Schemas/Microsoft.OrchestratorRecognizer.schema

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IRecognizer)",
+    "title": "Orchestrator Recognizer",
+    "description": "Orchestrator recognizer.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional unique id using with RecognizerSet."
+        },
+        "modelPath": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Model",
+            "description": "NLR model file path.",
+            "default": "settings.orchestrator.modelpath"
+        },
```

Comment:
should be "=settings.orchestrator.modelPath"
Also, the code should be settingit in 
   publlic StringExpression ModelPath {Get;set;} = "=settings.orchestrator.modelPath"

Reply:
updated. didnt set this in code since neither LUIS or QnA recognizer do this in code today. I'm not sure why but I suspect there was a reason why we are not doing it there.